polaris-react/src/styles/foundation/_focus-ring.scss

```diff
@@ -38,6 +38,14 @@
       transition: box-shadow var(--p-motion-duration-100) var(--p-motion-ease);
       border-radius: $border-radius;
       @content;
+
+      #{$se23} & {
+        top: calc(-1 * var(--p-space-025));
+        right: calc(-1 * var(--p-space-025));
+        bottom: calc(-1 * var(--p-space-025));
+        left: calc(-1 * var(--p-space-025));
+        border-radius: var(--p-border-radius-2);
+      }
```

Comment:
💯 Just went through that PR and cleaned it up. [So much easier using outline ](https://github.com/Shopify/polaris/pull/9317/files#diff-528b6953880285cf5db9183ed8fd17b779b3ccaca1db8f0785979139832308a2R28-R30)

I also just removed the transition value within the focus-ring so that we don't have to worry about adding that anymore: https://github.com/Shopify/polaris/pull/9336

Reply:
I'm inclined to leave this as is for now. It's one change to the focus ring in one place. If I update button to use outline, I'm looking at making that change in 9 places behind the feature flag 9 times. We for sure need to clean this up, but let's do it after this big push.